packages/backend/index.js

```diff
@@ -78,8 +161,46 @@ app.get("/reviews/", async (req, res) => {
   }
 });
 
+// creating a review
+app.post("/review/:id", async (req, res) => {
+  try {
+    const reviewToAdd = req.body;
+    console.log(reviewToAdd);
+    // get the user of the person trying to write review
+    // note: the username is gotten from the frontend which isn't secure, to update once we get auth running
```

Comment:
so just to clarify my understanding for the comment, the reviews route is missing the implementation of authenticating users properly with tokens, but once that step is finished we would then implement the autheticateUsers function here as well? (sorry im not super familiar with this authenticating stuff haha)

Reply:
This comment is actually leftover from before my changes haha sorry for the confusion. i will update comments as well so the code is better documented for everyone who hasn't worked on the auth stuff